packages/beeq/src/components/date-picker/_storybook/bq-date-picker.stories.tsx

```diff
@@ -0,0 +1,193 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html, nothing } from 'lit-html';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import mdx from './bq-date-picker.mdx';
+import { INPUT_VALIDATION } from '../../input/bq-input.types';
+
+const meta: Meta = {
+  title: 'Components/Date picker',
+  component: 'bq-date-picker',
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
+  argTypes: {
+    autofocus: { control: 'boolean' },
+    'clear-button-label': { control: 'text' },
+    'disable-clear': { control: 'boolean' },
+    distance: { control: 'number' },
+    disabled: { control: 'boolean' },
+    form: { control: 'text' },
+    min: { control: 'text' },
+    max: { control: 'text' },
+    locale: { control: 'text' },
+    'show-outside-days': { control: 'boolean' },
+    'first-day-of-week': { control: 'number' },
+    name: { control: 'text' },
+    open: { control: 'boolean' },
+    'panel-height': { control: 'text' },
+    placement: {
+      control: 'select',
+      options: [
+        'top',
+        'top-start',
+        'top-end',
+        'bottom',
+        'bottom-start',
+        'bottom-end',
+        'right',
+        'right-start',
+        'right-end',
+        'left',
+        'left-start',
+        'left-end',
+      ],
+    },
+    placeholder: { control: 'text' },
+    required: { control: 'boolean' },
+    skidding: { control: 'number' },
+    strategy: { control: 'select', options: ['fixed', 'absolute'] },
+    'validation-status': { control: 'select', options: [...INPUT_VALIDATION] },
+    range: { control: 'boolean' },
+    multi: { control: 'boolean' },
+    months: { control: 'number' },
+    value: { control: 'text' },
+    // Events
+    bqBlur: { action: 'bqBlur' },
+    bqChange: { action: 'bqChange' },
+    bqClear: { action: 'bqClear' },
+    bqFocus: { action: 'bqFocus' },
+    bqInput: { action: 'bqInput' },
+    // Not part of the public API, so we don't want to expose it in the docs
+    noLabel: { control: 'boolean', table: { disable: true } },
+    prefix: { control: 'boolean', table: { disable: true } },
+    suffix: { control: 'boolean', table: { disable: true } },
+    customDisallowedDate: { control: 'text' },
```

Comment:
I cannot see this prop being defined and used in the implementation of the component (_in bq-date-picker.tsx_) 🤔